core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -86,6 +87,24 @@ object Resource extends ConfigFormat {
     }
   }
 
+  final class DMGFile(val path: Path) extends IntellijResource {
+    def installTo(target: Path): Unit = {
+      import sys.process._
+
+      // attach disk image from .dmg file to local filesystem
+      val dmgDir = s"${System.getProperty("java.io.tmpdir")}dmg_dir"
```

Comment:
Nit: is there a guarantee that `java.io.tmpdir` ends with a `/`? I'd rather trim the trailing path separator if present and then append it explicitly, or best use use path joining mechanism (like Python's `os.path.join`)

Reply:
Sure, switched to standard `resolve` method from `java.nio.file.Path` class